HIP/hip-28.md

```diff
@@ -348,11 +363,15 @@ The following is the set of terms and definitions used to define Policy Actions
 |`children`|Defines a list of Blocks that are grouped into a Policy Workflow|List of Blocks, see also example given below|
 | `defaultActive` | Determines if Policy Workflow or a Policy Action can be executed | `true`                                           |
 | `permissions`   | Security Policy associated with a specific role                 | `ROOT_AUTHORITY`                                 |
+| `Roles`   | Available roles from which the user can choose             | `Installer`                                 |
```

Comment:
Should these be capital letters (as in `Roles` vs `roles`, same for other changes in the table)

Reply:
I think these need to be lower-case, will correct - thank you!